pkg/proc.attr.hydfab/R/proc_attr_grabber.R

```diff
@@ -636,3 +637,75 @@ grab_attrs_datasets_fsds_wrap <- function(Retr_Params,lyrs="network",overwrite=F
 
   return(ls_comids_all)
 }
+
+check_attr_selection <- function(attr_cfg_path = NULL, vars = NULL){
+  # Read in the menu of attributes available through FSDS
+  attr_menu <- yaml::read_yaml('./fsds_attr_menu.yaml')
+  
+  if(!is.null(attr_cfg_path)){
+
+    # Read in the user defined config of attributes of interest
+    # attr_cfg_path <- '../../scripts/eval_ingest/xssa/xssa_attr_config.yaml'
+    attr_cfg <- yaml::read_yaml(attr_cfg_path)
+    
+    # Determine which data sets the user specifies attributes from
+    ha_vars_sel <- base::lapply(attr_cfg$attr_select, function(x) names(x)) %>%
```

Comment:
Note that the above code chunk needs some object name changes to jive with the existing names of objects in this function

Reply:
I took your advise to make these functions dataset agnostic, though note that the final product is called vars_sel rather than vars_ls, so you would do: 
`Retr_Params$vars <- vars_sel`

Reply:
See new reporting from `verbose = TRUE` which reports which variables datasets are being queried to the user.